examples/README.md

```diff
@@ -1,4 +1,68 @@
-# Doorman Integration Examples
+# Integration Examples
 
 - [Python / Flask](python/): A Web UI interacts with Auth0 and a Flask API
 
+# Configuration Examples
+
+## Superusers with Doorman tags
+
```

Comment:
I think the documentation here needs a "story". 
In the case of Symbols the story is:
> "For example, use Doorman to maintain a carefully curated list of people who should become "superusers" when they log in to a certain service. This means the service doesn't have build functionality to promote and demote superusers."

The basic story is that you can hard code a list of users and or groups that should get a certain "tag". And you can query the `/allowed` endpoint to then question if the user has this tag or not.